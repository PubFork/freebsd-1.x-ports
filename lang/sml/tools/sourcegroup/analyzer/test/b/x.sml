structure B = struct
  local structure C = struct val z = A.x end
        structure A = struct val x = 4 val q = 7 end
  in
    val y = A.q
  end
end
