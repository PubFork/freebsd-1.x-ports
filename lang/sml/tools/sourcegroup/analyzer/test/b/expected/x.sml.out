source sml "b/x.sml"
  import structure A 
  export structure B ;

